ipyleaflet/leaflet.py

```diff
@@ -1395,6 +1395,8 @@ class WidgetControl(Control):
     max_height = Int(default_value=None, allow_none=True).tag(sync=True)
     min_height = Int(default_value=None, allow_none=True).tag(sync=True)
 
+    bg_transparent = Bool(False).tag(sync=True, o=True)
```

Comment:
Maybe `transparent_bg`?
Why is it an option, unlike the other attributes?

Reply:
Because it needs to get pulled when `get_options` is called over here
`this.obj = L.control.widgetcontrol(this.get_options());`

Reply:
So that I can later access it as an option ove here
```
onAdd(map) {
    if(this.options.bgTransparent)
      this._container = L.DomUtil.create('div');
    else
      this._container = L.DomUtil.create('div', 'leaflet-widgetcontrol');
```

Reply:
This is not an option, you should not use it as an option. But you can access it directly from the model doing:
```javascript
onAdd(map) {
    if(this.model.get('bg_transparent'))
      this._container = L.DomUtil.create('div');
    else
      this._container = L.DomUtil.create('div', 'leaflet-widgetcontrol');
```

Reply:
Ok Cool

Reply:
But I don’t think the model is accessible in onAdd see below
![image](https://user-images.githubusercontent.com/30625612/96335737-45612580-1098-11eb-8e4f-524e08e2f1da.png)

Reply:
Oh yes, I see now.

I forgot we had a custom Control class for this, then I guess we can make it an official option indeed. But then we might want to do like in Leaflet Controls and officially define it as an option: https://github.com/Leaflet/Leaflet/blob/master/src/control/Control.Zoom.js#L17-L36
This might not be required for it to work but I think it's cleaner.

Reply:
How to define it as an option like, because this https://github.com/Leaflet/Leaflet/blob/master/src/control/Control.Zoom.js#L17-L36 syntax is not working with the way currently the class is defined and also  I am not a JS expert 😬

Reply:
Actually, this seems to only be the default value of the options attribute. Ok let's not bother about this.